curriculum/challenges/english/25-front-end-development/workshop-fruit-search-app/67ed044dc76cf4a441618f2c.md

```diff
@@ -0,0 +1,137 @@
+---
+id: 67ed044dc76cf4a441618f2c
+title: Step 5
+challengeType: 0
+dashedName: step-5
+---
+
+# --description--
+
+Create a `query` state variable with an empty string (`''`) value and its setter function, `setQuery`, using the `useState` Hook.
```

Comment:
You need to provide context on why they are doing this. otherwise, campers won't know what they are doing or why they are doing it.

```suggestion
Now that the basic JSX is set up, it's time to track what the user types into the search input field. In previous lecture videos, you learned about controlled inputs, where form elements are tied directly to state variables in React.

To begin, create a state variable named `query` with an initial value of an empty string (''). Also define its corresponding setter function, `setQuery`, using the `useState` Hook.
```